volumic_br80_ultra.xml.fdm_material

```diff
@@ -0,0 +1,28 @@
+<?xml version='1.0' encoding='utf-8'?>
+<fdmmaterial version="1.3" xmlns="http://www.ultimaker.com/material">
+  <metadata>
+    <name>
+      <brand>Volumic</brand>
+      <material>METAL</material>

```

Comment:
Similarly, if it is not an abbreviation, it is capitalised normally.
```suggestion
      <material>Metal</material>
```

Reply:
ok, fixed